lib/bitmovin/encoding/encodings/list.rb

```diff
@@ -9,25 +9,29 @@ def init(path, klass)
       end
       attr_reader :resource_path
       attr_reader :klass
+      attr_reader :items
     end
 
     attr_accessor :encoding_id
     def initialize(encoding_id)
       @encoding_id = encoding_id
+      @items = []
     end
 
     def list(limit = 100, offset = 0)
       path = File.join("/v1/encoding/encodings/", @encoding_id, self.class.resource_path)
       response = Bitmovin.client.get(path, { limit: limit, offset: offset })
-      result(response)['items'].map { |item| self.class.klass.new(@encoding_id, item) }
+      @items = result(response)['items'].map { |item| self.class.klass.new(@encoding_id, item) }
```

Comment:
This overwrites the `@items` array on every call?
Are you using the `@items` array as the cache and the `list()` method to populate the cache? Because I think this should better be done explicitly?

Reply:
I am using the @items array as the cache and the list() method to populate the cache. The idea is to have at least a cached item but after thinking about it, we can revert the changes to this file as its unrelated this ticket

Reply:
Cool. everything else looks great.. I'll think about how caching might look like here..